authentication/typescript/azure/package.json

```diff
@@ -1,44 +1,48 @@
 {
-    "name": "@microsoft/kiota-authentication-azure",
-    "version": "1.0.6",
-    "description": "Authentication provider for Kiota using Azure Identity",
-    "main": "dist/cjs/index.js",
-    "module": "dist/es/index.js",
-    "types": "dist/cjs/index.d.ts",
-    "scripts": {
-        "build": "npm run build:cjs && npm run build:esm",
-        "build:cjs": "tsc -p tsconfig.cjs.json",
-        "build:esm": "tsc -p tsconfig.es.json",
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "clean": "rm -r ./dist"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git://github.com/microsoft/kiota.git"
-    },
-    "keywords": [
-        "Kiota",
-        "OpenApi",
-        "Azure",
-        "Authentication",
-        "OAuth"
-    ],
-    "author": "Microsoft",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/microsoft/kiota/issues"
-    },
-    "homepage": "https://github.com/microsoft/kiota#readme",
-    "publishConfig": {
-        "registry": "https://npm.pkg.github.com"
-    },
-    "dependencies": {
-        "tslib": "^2.3.1",
-        "@azure/core-auth": "^1.3.2",
-        "@microsoft/kiota-abstractions": "^1.0.27"
-    },
-    "devDependencies": {
-        "@types/node": "^17.0.13",
-        "typescript": "^4.5.5"
-    }
+  "name": "@microsoft/kiota-authentication-azure",
+  "version": "1.0.6",
+  "description": "Authentication provider for Kiota using Azure Identity",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
```

Comment:
This would change to dist/src/* because we are adding tests in this file. 
And the tsconfig.json would include test and src folders. This can be done in the version bump PR after abstraction dependencies are updated. 

Let me know if you need help with this.

Reply:
I'd rather tackle all the merging things I might have missed here so we don't forget anything.
Could you be more specific and use the suggestion tool please?